vagrant/Install-on-Ubuntu-22.sh

```diff
@@ -248,7 +248,8 @@ else                                  #DOCS:
     sudo systemctl restart apache2
 fi                                    #DOCS:
 
-# The Nominatim API is now available at `http://localhost/nominatim/`.
+# The Nominatim API is now available at `http://localhost/nominatim/`. To test if everything
+# works, point your browser to the status page at http://localhost/nominatim/status."
```

Comment:
You copied a '"' too much.

And let's get rid of the final full stop as well. I can already see the bug reports complaining about the 404 on `status.`. Or should we simply make it a markdown link to avoid copy mistakes?